developer_setup.md

```diff
@@ -13,145 +13,140 @@
+   | brew | `brew install openssl` |
 
-In order to compile Ruby, native Gems and native NodeJS modules, other libraries are required.
+3. In addition, in order to compile Ruby, native Gems and native NodeJS modules, other libraries are required.
 
-|      |     |
-| ---- | --- |
-| dnf  | `sudo dnf -y install @c-development libffi-devel postgresql-devel libxml2-devel libcurl-devel cmake python` |
-| yum  | `sudo yum -y install @development libffi-devel postgresql-devel libxml2-devel libcurl-devel cmake python` |
-| apt  | `sudo apt -y install build-essential libffi-dev libpq-dev libxml2-dev libcurl4-openssl-dev cmake python` |
-| brew | `brew install cmake` |
+   |      |     |
+   | ---- | --- |
+   | dnf  | `sudo dnf -y install @c-development libffi-devel postgresql-devel libxml2-devel libcurl-devel cmake python` |
+   | yum  | `sudo yum -y install @development libffi-devel postgresql-devel libxml2-devel libcurl-devel cmake python` |
+   | apt  | `sudo apt -y install build-essential libffi-dev libpq-dev libxml2-dev libcurl4-openssl-dev cmake python` |
+   | brew | `brew install cmake` |
 
-### Prerequisite services
+### Services
 
 ManageIQ requires a memcached instance for session caching and a PostgreSQL database for persistent data storage.
 
 #### memcached
 
-##### Install
+1. Install
 
-|            |     |
-| ---------- | --- |
-| dnf        | `sudo dnf -y install memcached` |
-| yum        | `sudo yum -y install memcached` |
-| apt        | `sudo apt -y install memcached` |
-| brew       | `brew install memcached` |
-| containers | N/A |
+   |            |     |
+   | ---------- | --- |
+   | dnf        | `sudo dnf -y install memcached` |
+   | yum        | `sudo yum -y install memcached` |
+   | apt        | `sudo apt -y install memcached` |
+   | brew       | `brew install memcached` |
+   | containers | N/A |
 
-##### Start the service
+2. Start the service
 
-|            |     |
-| ---------- | --- |
-| systemd    | `systemctl enable --now memcached` |
-| brew       | `brew services start memcached` |
-| containers | `podman run --detach --publish 11211:11211 memcached` |
+   |            |     |
+   | ---------- | --- |
+   | systemd    | `systemctl enable --now memcached` |
+   | brew       | `brew services start memcached` |
+   | containers | `podman run --detach --publish 11211:11211 memcached` |
 
 #### PostgreSQL
 
-##### Install
+1. Install
 
-|            |     |
-| ---------- | --- |
-| dnf        | `sudo dnf -y install postgresql-server` |
-| yum        | `sudo yum -y install postgresql-server` |
-| apt        | `sudo apt -y install postgresql` |
-| brew       | `brew install postgresql` |
-| containers | N/A |
-
-###### NOTE
+   |            |     |
+   | ---------- | --- |
+   | dnf        | `sudo dnf -y install postgresql-server` |
+   | yum        | `sudo yum -y install postgresql-server` |
+   | apt        | `sudo apt -y install postgresql` |
+   | brew       | `brew install postgresql` |
```

Comment:
I wonder if it makes sense to instead suggest installing `postgresql@10` `brew`, and equivalent for `dnf`/`yum`/`apt`.  Not sure if it is worth doing in this PR, but it is what I do on my end to stay consistent with the version we use.

Reply:
I too peg the version - I like nick's suggestion

Reply:
yeah, I was debating that one myself.  I'm not sure how to peg the version in other systems.  @bdunne @agrare ?

Reply:
I updated this one, but would like to update them all if possible.

Reply:
That is why I was kinda of suggesting solving this another time.  I don't have familiarity with the other package managers like I do with `brew`, so I couldn't provide a solid answer, and I figured we should be consistent with this documentation.

Reply:
Debian/Ubuntu it would be `apt install postgresql-10`

It depends on the version of your distro though, for example the latest stable version of debian `postgresql-11` is the oldest version available and I have `postgresql-12` and `postgresql-13` installed.  You would have to enable the prior version's apt source to get `apt install postgresql-10` to work.

Reply:
Yeah this is why we avoided the version numbers 😆

Reply:
Similar for Fedora, `dnf install postgresql-11` and 11 is the oldest version available on Fedora 33

Reply:
I can just put 11 across the board in the docs.

Reply:
> Yeah this is why we avoided the version numbers laughing

Haha yes, for devs it is probably better to have MIQ work with the latest version of PG (e.g. whatever `apt install postgresql` would bring in) even if it isn't "supported"

Reply:
Thanks updated.  I also added a Min/Max to the initial table of versions, since installing 11 would be weird otherwise.

Reply:
@carbonin has suggested using https://www.postgresql.org/download/linux/redhat/ in the past